spec/factories/users.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+    factory :user do
+         username {"usdfderiname"}
+         email    {"ussfsfernkiame@gmail.com"}
```

Comment:
Try adding sensible fake data like the followings:

username: johndoe
email: john@example.com